src/screens/setup/recover/RecoverWallet.js

```diff
@@ -42,12 +42,20 @@ class RecoverWallet extends Component {
       try {
         // const wallet = new ethers.Wallet('0x923ed0eca1cee12c1c3cf7b8965fef00a2aa106124688a48d925a778315bb0e5');
         // wallet.provider = provider;
-        let mnemonic, wallet;
-        mnemonic = this.state.mnemonic.trim();
-        wallet = ethers.Wallet.fromMnemonic(mnemonic);
-        wallet.provider = provider;
-        this.props.newWalletCreation(wallet); //pass state to redux to save it
-        this.props.navigation.dispatch(navigateToTokens);
+        if (this.props.debugMode === true) {
```

Comment:
use `if (__DEV__) {...}`

 `__DEV__` is a global variable in the JS runtime environment that is triggered too `true` when the package manager is running.

If there are other instances of this debugMode Please remove.

Reply:
Dam I did not know that. I'll create another pr for it